src/test/kotlin/ua/marchenko/artauction/auth/service/AuthServiceTest.kt

```diff
@@ -5,77 +5,85 @@ import org.junit.jupiter.api.Assertions.assertEquals
-import org.mockito.Mockito.verify
-import org.mockito.kotlin.doReturn
-import org.mockito.kotlin.doThrow
-import org.mockito.kotlin.whenever
 import ua.marchenko.artauction.auth.controller.dto.AuthenticationRequest
 import ua.marchenko.artauction.auth.controller.dto.RegistrationRequest
+import ua.marchenko.artauction.auth.jwt.JwtServiceImpl
 
 class AuthServiceTest {
 
-    private val mockAuthManager: AuthenticationManager = mock(AuthenticationManager::class.java)
-    private val mockUserDetailsService: UserDetailsService = mock(UserDetailsService::class.java)
-    private val mockJwtService: JwtService = mock(JwtService::class.java)
-    private val mockUserRepository: UserRepository = mock(UserRepository::class.java)
-    private val mockPasswordEncoder: PasswordEncoder = mock(PasswordEncoder::class.java)
+    @MockK
+    private lateinit var mockAuthManager: AuthenticationManager
 
-    private val authenticationService: AuthenticationService = AuthenticationServiceImpl(
-        mockAuthManager,
-        mockUserDetailsService,
-        mockJwtService,
-        mockUserRepository,
-        mockPasswordEncoder
-    )
+    @MockK
+    private lateinit var mockUserDetailsService: CustomUserDetailsServiceImpl
+
+    @MockK
+    private lateinit var mockJwtService: JwtServiceImpl
+
+    @MockK
+    private lateinit var mockUserRepository: UserRepository
+
+    @MockK
+    private lateinit var mockPasswordEncoder: PasswordEncoder
+
+    @InjectMockKs
+    private lateinit var authenticationService: AuthenticationServiceImpl
+
+    @BeforeEach
+    fun setUp() {
+        MockKAnnotations.init(this)
+    }
 
     @Test
-    fun `authentication should return AuthenticationResponse if provided credentials are correct`() {
+    fun `should return AuthenticationResponse when provided credentials are correct`() {
         //GIVEN
         val authenticationRequest = AuthenticationRequest.random()
         val usernamePasswordAuthenticationToken =
             UsernamePasswordAuthenticationToken(authenticationRequest.email, authenticationRequest.password)
         val userDetails = CustomUserDetails(getRandomString(), getRandomString(), Role.ARTIST)
         val expectedResponse = AuthenticationResponse(getRandomString())
 
-        whenever(mockUserDetailsService.loadUserByUsername(authenticationRequest.email)) doReturn (userDetails)
-        whenever(mockJwtService.generate(userDetails)) doReturn (expectedResponse.accessToken)
+        every { mockAuthManager.authenticate(usernamePasswordAuthenticationToken) } returns (null)
+        every { mockUserDetailsService.loadUserByUsername(authenticationRequest.email) } returns (userDetails)
+        every { mockJwtService.generate(userDetails) } returns (expectedResponse.accessToken)
 
         //WHEN
         val result = authenticationService.authentication(authenticationRequest)
 
         //THEN
-        verify(mockAuthManager).authenticate(usernamePasswordAuthenticationToken)
```

Comment:
instead of deleting verification, replace it with mockk verification

Reply:
done